pillar.example

```diff
@@ -4,9 +4,7 @@ vmbuilder:
     vgname: "hypervisor_vg"
     lvm:
       - vgname: "hypervisor_vg"
-        devices: 
-          - "/dev/sdb5"
-          - "/dev/sda"
+        devices: "/dev/sdb5,/dev/sda"
```

Comment:
Why make this change? Was there an issue with leaving it as a list instead of using string.split(',') later?